roadmap.md

```diff
@@ -0,0 +1,38 @@
+# Usage of Epics
+An Epic is a grouping of smaller related issues/tasks that fits into a larger goal. An Epic can potentially span across multiple iterations. 
+
+To track how smaller issues fit into the larger goal, we use [task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists).
+Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. 
+It also leads to smaller pull requests, which are easier to review.
+
+>NOTE: To create an epic on Github, use the available Epic template.
+
+# Usage of Github issue template/forms
+We have [customized templates](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/blob/main/.github/ISSUE_TEMPLATE/epic.md) that helps contributors in 
+creating new issues in the repository. When creating an issue, choose the template that best fits the issue you are about to create.  
+
+# Communicate
+- Use `@mentions` to alert a person or entire team about a the issue. 
+- Use appropriate labels during the creation or the triaging of the issue. Labels help to categorise issues thus improving their discoverbility. Check out the existing labels on - https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/labels.  
+- Assign collaborators to issues to communicate responsibility.
+
+>NOTE: You can always reach out to the wider team via the available [communication channels](https://github.com/bf2fc6cc711aee1a0c2a/ffm-project/wiki/How-to-communicate-in-the-project) 
+
+# View the Roadmap
+The roadmap is defined in the [project view](https://github.com/orgs/bf2fc6cc711aee1a0c2a/projects/5) 
+We have two main views:
+- The roject view containing all the issues that have been triaged and potentially assigned to a Sprint.
+- The Epic view, containing larger goals/tasks.
+
+The two views are the source of truth and they are kept up to date to showing our roadmap.
+
+## How to add an issue to the Roadmap (TODO)
+
+## How to add an Epic to the Roadmap (TODO)
+
+# Usage of Milestone
```

Comment:
I would much prefer to use the `Sprint` field in the project view to assign to a sprint rather than the milestone.

This is a custom iteration type field created in the project view and can only be assigned in the project view.

Reply:
Yes, the milestone are aligned to a quarterly "release". An issue could be assigned to a milestone and a sprint (using the iteration field) at the same time. My thinking is, an issue may as well be completed within a sprint which in turn is part of an even bigger / wider milestone.

Reply:
Perfect, that sounds good to me. I am not sold on the idea of tying milestones to a quarter but the text is good.

Can you add a section on Sprints and using that field as well, please. Even if it is to say that the sprint field should generally be assigned during sprint planning.

Reply:
> Perfect, that sounds good to me. I am not sold on the idea of tying milestones to a quarter but the text is good.

I think how we define milestones will be tied to our release process. If we say that we'll be doing quarterly release (creating a new version), that means a release each 3 months, to be that sounds like less frequent. I am fine too with keeping the text as it is now and refine it as we go.  

> Can you add a section on Sprints and using that field as well, please. Even if it is to say that the sprint field should generally be assigned during sprint planning.

Good point. I'll add it. I remember having mentioned Sprint without clearly saying how it is defined.